extensions/spatie-documentation/src/laravel-backup.tsx

```diff
@@ -0,0 +1,167 @@
+    return algoliaClient.initIndex(INDEX);
+  }, [algoliaClient, INDEX]);
+
+  const [searchResults, setSearchResults] = useState<any[] | undefined>();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const hierarchyToArray = (hierarchy: KeyValueHierarchy) => {
+    return Object.values(hierarchy)
+      .filter((hierarchyEntry: string | unknown) => hierarchyEntry)
+      .map((hierarchyEntry: string) => hierarchyEntry.replace("&amp;", "&"));
+  };
+
+  const getTitle = (hit: SpatieLaravelBackupDocsHit): string => {
+    return hierarchyToArray(hit.hierarchy).pop() || "";
+  };
+
+  const getSubTitle = (hit: SpatieLaravelBackupDocsHit): string => {
+    const highlightResult = striptags(hit._highlightResult?.content?.value || "");
+    if (highlightResult) {
+      return highlightResult;
+    }
+
+    const hierarchy = hierarchyToArray(hit.hierarchy) || [];
+    hierarchy.pop();
+    return hierarchy.join(" > ");
+  };
+
+  const search = async (query = "") => {
+    if (query === "") {
+      return;
+    }
+    setIsLoading(true);
+
+    return await algoliaIndex
+      .search(query, {
+        hitsPerPage: 11,
+        facetFilters: ["version:" + getPreference.spatieLaravelBackupVersion, "project:laravel-backup"],
+      })
+      .then((res) => {
+        setIsLoading(false);
+        return res.hits;
+      })
+      .catch((err) => {
+        setIsLoading(false);
+        showToast(ToastStyle.Failure, "Error searching Spatie Laravel Backup Documentation", err.message);
+        return [];
+      });
+  };
+
+  useEffect(() => {
+    (async () => setSearchResults(await search()))();
+  }, []);
+
+  const currentDocs = documentation[getPreference.spatieLaravelBackupVersion];
+  return (
+    <List
+      throttle={true}
+      isLoading={isLoading}
+      onSearchTextChange={async (query) => setSearchResults(await search(query))}
+    >
+      {searchResults?.map((hit: SpatieLaravelBackupDocsHit) => {
+        return (
+          <List.Item
+            key={hit.objectID}
+            title={getTitle(hit)}
+            subtitle={getSubTitle(hit)}
+            icon="spatie-icon.png"
+            actions={
+              <ActionPanel title={hit.url}>
+                <OpenInBrowserAction url={hit.url} title="Open in Browser" />
+                <CopyToClipboardAction content={hit.url} title="Copy URL" />
+              </ActionPanel>
+            }
+          />
+        );
+      }) ||
+        Object.entries(currentDocs).map(([section, items]: Array<any>) => {
+          return (
+            <List.Section title={section} key={section}>
+              {items.map((item: any) => {
+                return (
+                  <List.Item
+                    key={item.url}
+                    title={item.title}
+                    icon="spatie-icon.png"
+                    actions={
+                      <ActionPanel title={item.url}>
+                        <OpenInBrowserAction url={item.url} title="Open in Browser" />
```

Comment:
```suggestion
                <OpenInBrowserAction url={hit.url} />
```
Redundant since it's the default value